### ISSUE TITLE
t1ger_bankrobbery qb?

### ISSUE THREAD

**Issue by ItssJxstnDe** [t1ger-scripts/t1ger-mix-bundle#1]
YOu Mind to make a qb version?

**Comment by t1ger-scripts** [comment]
> YOu Mind to make a qb version?

Nope. But anyone who is interested is more than welcome to do so and make a PR and I will merge it.
